v2.2/follower-reads.md

```diff
@@ -0,0 +1,108 @@
+
+{{site.data.alerts.callout_danger}}
+The Follower Read feature is only available to [enterprise](https://www.cockroachlabs.com/product/cockroachdb/) users.
+{{site.data.alerts.end}}
+
+## What are Follower reads?
+
+Follower reads are a mechanism to let any replica of a range serve a read request, but are only available for read queries that are sufficiently in the past, i.e., using `AS OF SYSTEM TIME`. Currently, follower reads are available for any read operation at least 48 seconds in the past, though there is active work reduce that window. 
+
+To future-proof this feature in your code, we've also included a convenience function that runs your queries at a time as close as possible to the present time while remaining safe for follower reads.
+
+In widely distributed deployments, using follower reads can reduce the latency of read operations (which can also increase throughput) by letting the replica closest to the gateway serve the request, instead of forcing the gateway to communicate with the leaseholder, which could be geographically distant.
+
+## Settings
+
+### Enable/disable Follower reads
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.closed_timestamp.follower_reads_enabled` to:
+
+- `false` to disable follower reads _(default)_
+- `true` to enable follower reads 
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET CLUSTER SETTING kv.closed_timestamp.follower_reads_enabled = true;
+~~~
+
+#### When to use Follower reads
+
+Follower reads return consistent historical reads; currently a minimum of 48 seconds in the past, though we are actively working on reducing that number.
+
+As long as your `SELECT` operations can tolerate slightly outdated data, Follower reads can reduce read latencies and increase throughput.
+
+#### When not to use Follower reads
+
+You should not use follower reads when you need up-to-date data.
+
+### Make Follower Read-compatible queries
+
+Any `SELECT` statement with an `AS OF SYSTEM TIME` value at least 48 seconds in the past can be served by any replica (i.e., can be a Follower Read).
+
+To simplify this calculation, we've added a convenience function that will always set the `AS OF SYSTEM TIME` value to the minimum required for follower reads, `experimental_follower_read_timestamp()`:
+
+``` sql
+SELECT ... FROM ... AS OF SYSTEM TIME experimental_follower_read_timestamp();
+```
+
+### Make Follower Read-compatible transactions
+
+You can set the `AS OF SYSTEM TIME` value for all operations in a read-only transaction:
+
+```sql
+BEGIN AS OF SYSTEM TIME experimental_follower_read_timestamp();
+
+SAVEPOINT cockroach_restart;
+
+SELECT ...
+SELECT ...
+
+COMMIT;
+```
+
+## How Follower reads works
```

Comment:
If "Follower Reads" is a proper noun and we want the plural variant, suggest this heading should be:

"How Follower Reads work"

(Same comment everywhere this term is used, so I don't annoy you with tons more comments. :-} )